tests/conftest.py

```diff
@@ -306,53 +303,94 @@ def connect_rpc_embedded_server(port: int):
     return client
 
 
+def _launch_mechanical_rpyc_server(rootdir: str, version: int):
+    """Start rpyc server process, return the process object."""
+    from ansys.mechanical.core.embedding.rpc.utils import get_free_port
+
+    server_py = os.path.join(rootdir, "tests", "scripts", "rpc_server_embedded.py")
```

Comment:
```suggestion
    server_py = Path(rootdir) / "tests" / "scripts" / "rpc_server_embedded.py"
```
Use `pathlib` instead of `os` if possible